src/components/breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,41 @@
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  PageBreadcrumb,
+} from '@patternfly/react-core';
+
+export function QuayBreadcrumb(props: BreadcrumbProps) {
+  return (
+    <PageBreadcrumb>
+      <Breadcrumb>
+        {props.data.map((object, i) => (
+          <BreadcrumbItem to={object.to} data-testid={object.id} key={i}>
+            {object.title}
+          </BreadcrumbItem>
+        ))}
+        <BreadcrumbItem
+          to={props.activeItem.to}
+          data-testid={props.activeItem.id}
+          isActive
+        >
+          {props.activeItem.title}
+        </BreadcrumbItem>
+      </Breadcrumb>
+    </PageBreadcrumb>
+  );
+}
+
+type BreadcrumbProps = {
+  // List of page breadcrumbs
+  data: {
+    title: string;
+    id: string;
+    to: string;
+  }[];
+  // Active breadcrumb
+  activeItem: {
```

Comment:
I don't think we need this prop, we can use the last entry in the `data` field as the active item

Reply:
Yes, we can. I was debating on that too. I'll add activeitem in the `data` props

Reply:
Another option I was thinking was can we auto-generate the breadcrumb based on the URL so for example you are under

`/organisation/<orgname>/repositories/<reponame>/tags/<tagname>`

you have all the info you need in the path itself. Can you think of any cases where this might not work?

Reply:
Interesting. I was reading a blog where they do something similar. I will see if we can implement it that way.